<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bootForm</class>
 <widget class="QWidget" name="bootForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lin_pathfile">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lab_info">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="button_toboot">
     <property name="text">
      <string>Перевести в режим</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Порт</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lab_adrModbas">
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lab_boot">
     <property name="text">
      <string>Загрузить в устройство</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="button_boot">
     <property name="text">
      <string>Прошить</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QSpinBox" name="spin_adrmodbus"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lab_tobott">
     <property name="text">
      <string>Режим boot</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lab_file">
     <property name="text">
      <string>Выбрать файл(.cod)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QPushButton" name="button_file">
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
